modules/tfm/tfm/boards/partition/flash_layout.h

```diff
@@ -126,6 +129,7 @@
 /* In this target the CMSIS driver requires only the offset from the base
  * address instead of the full memory address.
  */
+#ifdef FLASH_PS_AREA_OFFSET
```

Comment:
I just did it that way since this is the symbol that is used below. So that if you for some reason define that symbol based on something other than `PM_TFM_EXTRA_ADDRESS` the definition below will still be included.